style/js/shield_defs.js

```diff
@@ -458,6 +474,86 @@ export function loadShields(shieldImages) {
       bottom: 6,
     },
   };
+  shields["US:OH:HOC:Falls"] = roundedRectShield(
+    "white",
+    "#006747",
+    "black",
+    2,
+    1,
+    null
+  );
+  shields["US:OH:PER:Monday_Creek"] = roundedRectShield(
+    "#006747",
+    "black",
+    "black",
+    2,
+    1,
+    null
+  );
+  shields["US:OH:TUS:Salem"] = {
+    backgroundImage: [shieldImages.shield40_us_oh_tus_salem],
+    textColor: "black",
+    padding: {
+      left: 0,
+      right: 3,
+      top: 0,
+      bottom: 3,
+    },
+  };
+
+  // If a township's road shields have the same shape and colors as the surrounding county's road shields,
+  // add a banner to distinguish the township road shields from the more prominent county road shields.
+
+  [
+    ["ASD", "TWP"],
+    ["ATH", "Trimble"],
+    ["MED", "York"],
+    ["PAU", "Latty"],
+    ["PAU", "Washington"],
+    ["WAS", "Aurelius"],
+    ["WAS", "Salem"],
+  ].forEach(
+    (countyAndTownship) =>
+      (shields[`US:OH:${countyAndTownship[0]}:${countyAndTownship[1]}`] =
+        banneredShield(shields[`US:OH:${countyAndTownship[0]}`], ["TWP"]))
+  );
+  [
+    ["FAI", "Violet"],
+    ["HOL", "Berlin"],
+    ["HOL", "Clark"],
+    ["HOL", "Knox"],
+    ["HOL", "Monroe"], // No black border in reality, but a border is needed for contrast.
+    ["HOL", "Paint"],
+    ["HOL", "Salt_Creek"],
+    ["KNO", "Liberty"],
+    ["KNO", "Milford"],
+    ["LIC", "Jersey"],
+    ["LOG", "Harrison"],
+    ["LOG", "Jefferson"],
+    ["LOG", "Lake"],
+    ["LOG", "Liberty"],
+    ["LOG", "McArthur"],
+    ["LOG", "Miami"],
+    ["LOG", "Monroe"],
+    ["LOG", "Perry"],
+    ["LOG", "Richland"],
+    ["LOG", "Rushcreek"],
+    ["LOG", "Stokes"],
+    ["LOG", "Union"],
+    ["MRW", "Canaan"],
+    ["MRW", "Harmony"],
+    ["MRW", "South_Bloomfield"],
```

Comment:
South Bloomfield Township posts both white-on-black and black-on-white shields. A few other townships are similarly inconsistent. I went for the path of least resistance for these townships.